core/vm/jump_table.go

```diff
@@ -108,6 +108,7 @@ func newCancunInstructionSet() JumpTable {
 	enable1153(&instructionSet) // EIP-1153 "Transient Storage"
 	enable5656(&instructionSet) // EIP-5656 (MCOPY opcode)
 	enable6780(&instructionSet) // EIP-6780 SELFDESTRUCT only in same transaction
+	enable2970(&instructionSet) // EIP-2970 (ISSTATIC opcode)
```

Comment:
I don't think this is a good idea, it would be better to add new Hardfork e.g. `CCv1` (for _Common Core V1_) and enable this EIP under that hardfork :)